addons/mail/static/src/models/chatter/chatter.js

```diff
@@ -268,6 +268,14 @@ function factory(dependencies) {
         hasFollowers: attr({
             default: true,
         }),
+        /**
+         * Whether the message list should manage its scroll.
+         * In particular, when the chatter is on the form view's side,
+         * then the scroll is managed by the message list
```

Comment:
dot at the end.

If you give an example, I would also give the problematic one here: the message list should not manage its scroll when it's sharing it with the rest of the page.